nylas/client/restful_models.py

```diff
@@ -876,6 +877,72 @@ def as_json(self):
         return dct
 
 
+class Webhook(NylasAPIObject):
+    attrs = [
+        "id",
+        "callback_url",
+        "state",
+        "triggers",
+        "application_id",
+        "version",
+    ]
+    read_only_attrs = {"id", "application_id", "version"}
+
+    collection_name = "webhooks"
+    api_root = "a"
```

Comment:
Are collection_name and api_root used anywhere?

Reply:
@tjovicic yes! `collection_name` tells the class that makes API calls what the "endpoint" is.  Setting `api_root` tells the same API class that we should have `/a` prefix and to use the client ID in the outgoing URL. Setting these two variables results in the URL being formatted as `https://api.nylas.com/a/{client_id}/webhooks`.